trogon/trogon.py

```diff
@@ -23,6 +23,7 @@
     Footer,
 )
 from textual.widgets.tree import TreeNode
+import typer
```

Comment:
We don't want to force a dependancy on typer, for those who just want to use click...